### DISCUSSION TITLE
What concepts can be used to add or reorder multiple elements in a template without editing the template file?

### DISCUSSION THREAD

**Discussion by ingo-code** [pimcore/pimcore#14892] [Q&A / Support]
Can anyone help me by Answering this Question?
What concepts can be used to add or reorder multiple elements in a template without editing the template file?
Renderlets
 Blocks
 Areabricks
 Scheduled Blocks

I think Renderlets  Areabricks are correct.  
These are dynamic elements which editor can edit dynamic input. If I read that correctly, but these inputs don't make a correct result
 Neither does blocks and  Scheduled Blocks. Can someone help me?

**Comment by aweichler** [comment]
Areabricks and Blocks are correct
